drools-benchmarks/src/main/java/org/drools/benchmarks/turtle/runtime/KBaseCreationFromDTable1Benchmark.java

```diff
@@ -0,0 +1,54 @@
+package org.drools.benchmarks.turtle.runtime;
+
+import org.drools.benchmarks.common.util.RuntimeUtil;
+import org.drools.benchmarks.model.Account;
+import org.kie.api.runtime.KieSession;
+import org.kie.api.runtime.conf.ClockTypeOption;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.Level;
+import org.openjdk.jmh.annotations.Setup;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class KBaseCreationFromDTable1Benchmark extends AbstractSimpleRuntimeBenchmark {
+
+    private static int from = 999;
+    private static int to = 3102;
+
+    @Setup
+    public void addResources() {
+        addClassPathResource("kbase-creation/dtable1-kbase-creation.xls");
+    }
+
+    @Setup(Level.Iteration)
+    public void createKieSession() {
+        ksession = RuntimeUtil.createKieSession(kieBase, ClockTypeOption.get("pseudo"));
```

Comment:
I know we talked about extracting session creation outside the benchmark, but please remove this createKieSession() method and insertFactsAndFireAllRules() method, so the ones from the abstract class are used. If the session creation is part of the benchmark method in the abstract class (therefore in other benchmarks), we should use it the same way. What do you think?

Reply:
I agree. The main reason I override it was to catch how many rules were fired. But there's nothing more being done so there's no need to override it.